pkg/manifests/types.go

```diff
@@ -283,6 +283,10 @@ type NodeExporterConfig struct {
 	// For nodes with a high CPU count, you can set the limit to a low number, which  saves resources by preventing Go routines from being scheduled to run on all CPUs.
 	// However, I/O performance degrades if the `maxProcs` value is set too low, and there are many metrics to collect.
 	MaxProcs uint32 `json:"maxProcs,omitempty"`
+	// A list of network devices, as regular expressions, to be excluded from the relevant collectors configuration such as `netdev` and `netclass`.
+	// When not set, the Cluster Monitoring Operator uses a predefined list of devices to be excluded to minimize the impact on memory usage.
```

Comment:
should we detail that passing an empty list = exclude no interface?

Reply:
done